pkg/libvirttools/virtualization.go

```diff
@@ -371,6 +433,15 @@ func (v *VirtualizationTool) CreateContainer(in *kubeapi.CreateContainerRequest,
 		return "", err
 	}
 
+	// Comment out when [https://github.com/Mirantis/virtlet/issues/295] is resolved.
+	// (also import "k8s.io/kubernetes/pkg/kubelet/types")
+	// containerName := in.Config.Labels[types.KubernetesContainerNameLabel]
```

Comment:
`containerName` in this context is equal to `settings.domainName`, passed as first argument to `v.metadataStore.SetContainer`. This is one of values returned by CRI runtime, and then internally added by kubelet to labels on it's side. At this time kubelet do not knows it and so - can not pass it in its labels.